guides/v3.4.0/object-model/index.md

```diff
@@ -1,18 +1,21 @@
-You'll notice standard JavaScript class patterns and the new ES2015
-classes aren't widely used in Ember. Plain objects can still be found,
-and sometimes they're referred to as "hashes".
+One of the first things you'll notice when you generate JavaScript files in Ember is that most of the code you will write goes inside of an object. While an Ember Object might look a lot like an ES2015 JavaScript class, it has some special properties.
 
-JavaScript objects don't support the observation of property value changes.
-Consequently, if an object is going to participate in Ember's binding
-system you may see an `Ember.Object` instead of a plain object.
+### Introducing: Ember Objects
 
-[Ember.Object](https://www.emberjs.com/api/ember/release/modules/@ember%2Fobject) also provides a class system, supporting features like mixins
-and constructor methods. Some features in Ember's object model are not present in
-JavaScript classes or common patterns, but all are aligned as much as possible
-with the language and proposed additions.
+The [Ember Object](https://www.emberjs.com/api/ember/release/classes/EmberObject) class extends plain JavaScript objects to provide core framework functions such as participating in Ember's binding system.
 
-Ember also extends the JavaScript `Array` prototype with its
-[Ember.Enumerable](https://emberjs.com/api/ember/2.15/classes/Ember.Enumerable) interface to provide change observation for arrays.
+Most objects in Ember, including Routes, Models, Services, Mixins, Controllers, and Components extend the `EmberObject` class.
 
-Finally, Ember extends the `String` prototype with a few [formatting and
-localization methods](https://www.emberjs.com/api/ember/release/classes/String).
+### Why Ember Objects?
+
+The most important reason is that an Ember Object can be watched for changes – or _observed_. Ember Objects, being [observable](https://www.emberjs.com/api/ember/release/classes/Observable), supports observing any of its properties including [computed properties](https://guides.emberjs.com/release/object-model/computed-properties/). It is one of the fundamental ways that models, controllers and views communicate with each other in an Ember application.
```

Comment:
Possible simplification of "Ember Objects, being observable...."

For example, being [observable](https://www.emberjs.com/api/ember/release/classes/Observable) is important for [computed properties](https://guides.emberjs.com/release/object-model/computed-properties/).

Reply:
Fixed!